src/mono/mono/eglib/glib.h

```diff
@@ -1031,6 +1032,32 @@ size_t     g_utf16_len     (const gunichar2 *);
 #define u16to8(str) g_utf16_to_utf8(str, (glong)strlen(str), NULL, NULL, NULL)
 #endif
 
+typedef gpointer (*GConvertCustomAllocator) (gsize req_size, gpointer custom_alloc_data);
+
+typedef struct {
+	gpointer buffer;
+	gsize buffer_size;
+	gsize req_buffer_size;
+} GConvertDefaultCustomAllocatorData;
+
+static
+gpointer
+g_converter_default_custom_allocator_func (gsize req_size, gpointer custom_alloc_data)
```

Comment:
nit: it's not the default, the default is `NULL`.  This is some kind of fixed buffer custom allocator.

Reply:
Was moving back and forth on this, had fixed_buffer at one point, but then moved back to the default custom allocator, so not the default allocator, but maybe should use a different naming to make things even more clear.